pedrovgs/HaveANiceDay
# Docker

### :pushpin: References
* **Issue:** Add docker support

### :tophat: What is the goal?

Configure a ``docker-compose.yml`` file so we can easily run our application locally.

### How is it being implemented?

We've created a new folder named ``docker`` and inside we've placed a ``docker-compose.yml`` file initializing a MYSql 5.6 database and a elastic-mq service. In the future, if needed we will add a redis service. We've also updated the project documentation.

### How can it be tested?

Checkout this branch and execute:

 ```
cd docker 
docker-compose up 
```

You'll see how the queues and the mysql DB is initialized properly.
